src/main/java/blusunrize/immersiveengineering/common/blocks/IEBaseBlock.java

```diff
@@ -132,13 +134,19 @@ public boolean canRenderInLayer(BlockState state, BlockRenderLayer layer)
 
 	public IEBaseBlock setLightOpacity(int opacity)
 	{
+		// see getLightValue(state)
 		lightOpacity = opacity;
 		return this;
 	}
 
 	@Override
-	public int getLightValue(BlockState state, IWorldReader world, BlockPos pos)
+	public int getLightValue(BlockState state)
 	{
+		//TODO double check. getLightValue() is a more tricky one now (at least at the present time).
+		//  	 Light values are defined in the block properties at construction, and it seems that
+		//	 	 values returned here must be 0 or exactly match that light value. Vanilla blocks with
+		//		 switched light use something like: `return LIT ? super.getLightValue(state) : 0`.
+		//     -> So it might be good to drop `lightOpacity` and use the vanilla frame here.
```

Comment:
Isn't `lightOpacity` something different from `lightValue`? The first one changes how much light is blocked (glass vs wood), the second one how much is emitted (torch vs RS torch).

Reply:
Yes, I shuffled this. Pushed commit with the `getOpacity()` override.